app/assets/javascripts/modules/FilterTable.js

```diff
@@ -45,34 +46,49 @@ define(['jquery', 'DoughBaseComponent', 'List'],
   };
 
   /**
-   * makeFilterTargetClasses
+   * toggleFilterWrappers function
    *
-   * Find columns with filterTargetClass and apply individual classes
-   * to them, since this is what ListJS wants.
+   * show/hide fitlerWrappers when all filterTargetClasses are hidden
    *
-   * @return {Array} array of classes used
    */
-  FilterTableProto.makeFilterTargetClasses = function () {
-    var filterClasses = [];
+  FilterTableProto.toggleFilterWrappers = function() {
+  	var self = this;
 
-    this.$el.find('.' + this.config.filterTargetClass).each(function(_, cell) {
-      var $cell = $(cell),
-          filterClass = 'js-filterable-' + $cell.index();
-      $cell.addClass(filterClass);
-      filterClasses.push(filterClass);
-    });
+		$(self.config.filterWrapperClass).each( function(index, element ) {
+			self.toggleFilterWrapper( $(element) )
+		});
+	}
 
-    return $.unique(filterClasses);
+  /**
+   * toggleFilterWrapper function
+   *
+   * show/hide fitlerWrapper when all filterTargetClasses are hidden
+   *
+   */
+
+  FilterTableProto.toggleFilterWrapper = function(element) {
+		var list = element.find(this.config.filterListClass)
+		var rows = $(list[0]).find('tr')
+		var hidden_rows = $(rows).filter(function() {
+		  return $(this).css('display') == 'none';
+		});
```

Comment:
There are a few things here:

1. MAS style dictates that we don't have multiple `var`s in a scope. We wrap them all into one var statement, separated with commas.
2. I believe the style is also to use the `$` prefix on variables that contain jQuery elements (took me a while to get this — coming from Angular).
3. I'm not sure, but I don't think we need to do `$(list[0])` — we can use `list.first()`
4. MAS style dictates using camelCase rather than underscores, so `hidden_rows` -> `hiddenRows`.
5. We also don't need that extra jQuery wrapper on `$(rows)`.
6. If you like, I think you could use `$(rows).filter(':hidden')` instead of the filter. That's just an idea though — each approach has its merits.
7. `element` -> `filterWrapper` maybe? There might be a better name than filterWrapper — tableWrapper?